README.md

```diff
@@ -93,7 +115,31 @@ Your Answer:
 
 What are the differences between calling and referencing a function? Please provide examples of each.
 
+
+
 ```text
+Once function is defined,
+
+var addNumber = function(num1, num2) {
+    x = num1 + num2;
+    console.log(x);
+}
+
+1. you can call a function.(executes the function)
+  addNumber(2,3);
+   // showing 5
+
+2. you can referencing it.(show the details of the function)
+  addNumber
+  // showing
+  addNumber = function(num1, num2) {
+      x = num1 + num2;
+      console.log(x);
+  }
+
```

Comment:
:+1: Referencing a function doesn't actually show the details of it -- it just refers to the function without running it. It's like me saying "the microwave" versus actually turning on the microwave.